### ISSUE TITLE
First draft for trait to disease mappings

### ISSUE THREAD

**Issue by matentzn** [mapping-commons/mh_mapping_initiative#15]
In recent months we have migrated OBA to a modern infrastructure, so we should be able to create a draft mapping based on the following process:

- [ ] @rays22 Publish a VT-OBA mapping in SSSOM here and share with Elissa's team  [`oba_vt.sssom.tsv`]
- [ ] Elissa's team will make sure that all relevant VT codes are present
- [ ] @rays22 Merge OBA, HP and MP (robot merge) and runs the reasoner. Run a ROBOT (SPARQL) query to determine all VT-MP/HP  mappings (all of these are necessarily `narrow` from OBA to HP/MP!) [`oba_upheno.sssom.tsv`]
- [ ] @rays22 Send to share SSSOM mappings with Elissa's team
- [ ] @matentzn will share MONDO - OMIM mapping with Elissa's team
- [ ] Elissa's team will merge [HPOA](https://hpo.jax.org/app/download/annotation) annotations (HP to OMIM p2d's) with  [`oba_upheno.sssom.tsv`] and [`oba_vt.sssom.tsv`] to produce [`vt_mondo.sssom.tsv`, `vt_hp.sssom.tsv`, `vt_mp.sssom.tsv`] and provide feedback here
- [ ] If these "mappings" (not really mappings in the strict sense, but let's say vaguely mappings) are any good, then we are start thinking about adding other sources of p2d and _streamlining_ this process for other trait to disease use cases like GWAS

**Comment by dosumis** [comment]
@matentzn  - this all sounds reasonable.  Would you be able to prepare some slides/figures detailing each step? It would be especially helpful to fill in the gaps between reasoning and  SSSOM mapping files.

**Comment by matentzn** [comment]
Yeah, this is all a bit theoretical atm - we will create figures and slides for every step as it happens (I also need to wrap my head around the details) - the first one is a run of sssom-py that I will go through in our next Friday meeting.

**Comment by matentzn** [comment]
Also consider augmenting the MP-HP mappings with @sbello latest contribution:
https://github.com/mapping-commons/mh_mapping_initiative/pull/10

**Comment by matentzn** [comment]
While @obwalton is getting is a list of missing (unmapped) VT terms, here the world's first and entirely unreviewed set of phenotype-trait associations:

https://github.com/obophenotype/upheno-dev/blob/master/src/ontology/reports/phenotype_trait.sssom.tsv

@rays22

- [ ] Please review the first draft of phenotype-trait associations. 100% automated. Please let it be good! :D You can try to fit it all in a google sheet, but it is large.

**Comment by jamesamcl** [comment]
@matentzn Please could you explain the relationship between these two files:

https://github.com/obophenotype/upheno-dev/blob/master/src/ontology/reports/phenotype_trait.sssom.tsv mentioned above

https://raw.githubusercontent.com/obophenotype/upheno-dev/refs/heads/master/src/mappings/upheno-oba.sssom.tsv

**Comment by matentzn** [comment]
> @matentzn Please could you explain the relationship between these two files:

This file 

> https://github.com/obophenotype/upheno-dev/blob/master/src/ontology/reports/phenotype_trait.sssom.tsv 

is outdated and I now git rmed it. The other one is correct!
